doc/source/timedeltas.rst

```diff
@@ -109,6 +109,31 @@ The ``unit`` keyword argument specifies the unit of the Timedelta:
    to_timedelta(np.arange(5), unit='s')
    to_timedelta(np.arange(5), unit='d')
 
+timdelta limits
+~~~~~~~~~~~~~~~
+
+.. ipython:: python
+
+   # Timedeltas are represented as 64 bit integers. so the
+   # max in min integers determine the timedelta limits
+   min_int = np.iinfo(np.int64).min
+   max_int = np.iinfo(np.int64).max
+
+   # The smallest integer gives a NaT
+   pd.Timedelta(min_int)
+
+   # One more than the smallests integer gives the
```

Comment:
this is too much information, just show the ``pd.Timedelta(np.iinfo(np.int64).min+1)`` (with the NaT) explanation.